### ISSUE TITLE
[Cleanup] bot reports incorrect number of messages it mass deleted

### ISSUE THREAD

**Issue by owocado** [Cog-Creators/Red-DiscordBot#4155]
# Command bugs

#### Command name

[p]cleanup messages [>100]

#### What cog is this command from?

Cleanup

#### What were you expecting to happen?

Bot should report the exact or correct amount of messages it deleted from the given channel in console/logs.

#### What actually happened?

Bot reports incorrect amount of messages deleted in console. I think it passes the same number we give as input.

#### How can we reproduce this issue?

Load cleanup cog, if haven't.
Try to delete more than 100 messages in a given channel with `[p]cleanup messages`. Confirm when prompted to. 
Try to pass an integer larger than the total number of messages of last 2 weeks which exists in the target channel. <- This step is essential to reproduce this bug.
Observe in your console/logs the amount the bot returns as deleted messages.
![image](https://user-images.githubusercontent.com/24418520/89275927-94ceb280-d660-11ea-9bc9-2f6441c2a9c8.png)
(^^for context in this screenshot, I used `[p]cleanup messages 500` in a channel.)

Now check your guild audit logs and see how many messages the bot actually deleted and cross-verify with the output bot provided in console/logs.
![image](https://user-images.githubusercontent.com/24418520/89275865-7b2d6b00-d660-11ea-8933-29ecbd9d930d.png)
You should be able to confirm that bot reports incorrect number of messages it mass deleted.

-----
While I have only observed and tested this with `[p]cleanup messages` subcommand, I assume the same bug should occur with other cleanup subcommands becuase of how `check_100_plus()` staticmethod is handled, though I could be wrong, but please verify it with other `cleanup` subcommands.

**Comment by Vexed01** [comment]
I can verify it _is_ inaccurate with `[p]cleanup messages`.

`cleanup after` - working
`cleanup before` - working
`cleanup between`- working
`cleanup bot` - working (there is an unnecessary space after the number in the logs though)
`cleanup messages` - as you established, **not working**
`cleanup self` - working
`cleanup spam` - working
`cleanup text` - working
`cleanup user` - working

TLDR: Just cleanup messages

_Sigh_, too late it's already got a PR. Poor Vexed's testing server now having a destroyed message history...

**Comment by MeatyChunks** [comment]
I destroyed mine in the process too, don't worry! :D
